src/components/LoginView/LoginView.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+
+export const LoginView = ({ onLoggedIn }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = (event) => {
+    // this prevents the default behavior of the form, to reload the entire page
+    event.preventDefault();
+
+    const data = {
+      access: username,
+      secret: password
+    };
+
+    fetch("https://cf-movie-api.herokuapp.com/login", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(data)
```

Comment:
You should be sending json not strings.

Just like this
{
access: username,
  secret: password
}

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

It is a json string, not a string. The exercise content also recommends using json.stringify.